ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -28,13 +29,20 @@ module.exports = Backbone.AssociatedModel.extend({
         mouseLon: undefined,
         clickLat: undefined,
         clickLon: undefined,
+        dms: undefined,
         target: undefined,
         targetMetacard: undefined
     },
     updateClickCoordinates: function () {
+
+        var lat = this.get('mouseLat');
+        var lon = this.get('mouseLon');
+        var dms = ol.coordinate.toStringHDMS([lon, lat]);
```

Comment:
We're trying to move toward ES6 in new JavaScript code; `const` should be used instead of `var`.

Reply:
Fixed